nsds_lab_to_nwb/metadata/metadata_manager.py

```diff
@@ -1,13 +1,126 @@
+import logging
 import os
 import csv
 import pandas as pd
 from ..utils import (get_metadata_lib_path, get_stim_lib_path,
                      split_block_folder)
 
-from nsds_lab_to_nwb.common.io import read_yaml, csv_to_dict
+from nsds_lab_to_nwb.common.io import read_yaml, write_yaml, csv_to_dict
 
 
-_DEFAULT_EXPERIMENT_TYPE = 'auditory'  # for legacy sessions
+_DEFAULT_EXPERIMENT_TYPE = 'auditory'
+_WRITE_YAML_FILE = True     # export intermediate yaml files for metadata inspection
+                            # turn off when testing is over
```

Comment:
Testing is never over! :)
Maybe just have this be a kwarg for `MetadataReader`?

Reply:
I think it might be helpful to always write this out at the `save_path`

Reply:
Good point - now using `metadata_save_path` as a switch (save only if a path is provided)